sdk/test-utils/recorder/tsconfig.json

```
@@ -2,10 +2,8 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "declarationDir": "./types",
-    "target": "es2017",
     "outDir": "./dist-esm",
-    "lib": ["dom", "es5", "es6", "es7", "esnext"]
+    "lib": ["dom", "esnext"]
```

Do you use anything interesting from esnext?
```suggestion
    "lib": ["dom"]
```